docs/Chef-CLI-Catalog.md

```diff
@@ -0,0 +1,91 @@
+# Chef CLI (Catalog)
+
+Throughout the history of Chef, we have created and acquired a variety
+of tools that have grown to a point that, as a Chef practitioner, it is
+hard to know, discover and understand all of them. A few of the tools
+that our users use (almost) every day are:
+
+* chef (CLI that includes Policyfiles management)
+* hab (Habitat CLI)
+* inspec
+* ohai
+* kitchen (Test-kitchen)
+* knife
+* berks (Berkshelf)
+* cookstyle
+* delivery
+* chef-apply
+* chef-client
+* chef-run
+* chef-shell
+* chef-solo
+
+## Goals
+* Have a single, unified way to discover tools to interact with Chef products
+* Huge speed improvements
+* Gather data about how our users consume and interact with our tools (Telemetry)
+* Standardize the UX of our (CLI) tools
+* Have the flexibility to rename tools and commands within our ecosystem
+* Enable developers to use any supported programming languages at Chef for efficiency
+
+## Motivation
+
+    As a Chef practitioner,
+    I would like to have a single Command Line Interpreter (CLI) that lets me easily interact with every Chef product.
+    I would also like to have a way to discover new tools that can help me with my day to day work.
+    Finally, since I run these tools daily, I need them to run fast
+
+    As a Chef developer,
+    I would have a tremendous benefit from this architecture since
+    I would be able to gather information about how our users consume and interact with our tools,
+    as well as identifying the commands that are used the most,
+    so I can improve the speed and/or do extreme modifications to the code base (do a rewrite in a different, more efficient, programing language)
```

Comment:
This so states things you could do not why you'd want to. Is there a way to say this without talking about extreme changes. Anytime we use that kind of language it can alienate folks that want more measured changed/no change. It's ok if you really mean to drive extreme change and understand the above, but make sure that's the intent.